File: jcap-web/src/types/creditPackage.ts
export interface CreditPackage {
  id: number;
  name: string;
  credits: number;
  price: number;
  isActive: boolean;
  createdAt: string;
}

export interface CreateCreditPackageDto {
  name: string;
  credits: number;
  price: number;
}

export interface UpdateCreditPackageDto {
  name: string;
  credits: number;
  price: number;
  isActive: boolean;
}

File: Models/CreditPackage.cs
namespace JCAP.Models
{
    public class CreditPackage
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public int Credits { get; set; }
        public decimal Price { get; set; }       // VND
        public bool IsActive { get; set; } = true;
        public DateTime CreatedAt { get; set; }
    }

}
